google-cloud-tools-plugin/ultimate/src/com/google/cloud/tools/intellij/appengine/server/integration/AppEngineServerIntegration.java

```diff
@@ -48,4 +55,9 @@ public ApplicationServerPersistentDataEditor createNewServerEditor() {
     // project wizard because there is a special 'Google App Engine' option
     return null;
   }
+
+  @Override
+  public ApplicationServerHelper getApplicationServerHelper() {
```

Comment:
Yeah I saw that, but thought I'd use the helper to name the application servers, e.g., App Engine Local Server. Otherwise they show up as Generic